### ISSUE TITLE
Long multi-line comments produce false positive long line lint

### ISSUE THREAD

**Issue by pwildenhain** [alanmcruickshank/sqlfluff#236]
I've downloaded the most recent version from GitHub :octocat: ✔️ 

Reprex: test.sql
```sql
select
    my_field
from
    /* this is a long multi-line comment that seeks
    to explain why we are selecting from this table
    rather than another table
    */
    my_table

```

```
$ sqlfluff lint test.sql
== [test.sql] FAIL
L:   7 | P:   7 | L016 | Line is too long.
```

Using one-line comments does **not** result in the same warnings

```sql
select
    my_field
from
    -- this is a long multi-line comment that seeks
    -- to explain why we are selecting from this table
    -- rather than another table
    my_table
```

**Comment by alanmcruickshank** [comment]
This is a great issue report, and also flagged up quite a few underlying issues in the handling of multiline comments. Thanks! 🙌 

PR coming shortly, would love your feedback in testing the new functionality.
